modules/perception/fusion/lib/data_fusion/existance_fusion/dst_existance_fusion/dst_existance_fusion.cc

```diff
@@ -286,7 +286,7 @@ void DstExistanceFusion::UpdateToicWithoutCameraMeasurement(
                         {ToicDstMaps::TOICUNKOWN, dist_score}});
   // TODO(yuantingrong): hard code for fused toic bba
   const double toic_fused_w = 1.0;
-  fused_toic_ = fused_toic_ + toic_evidence * in_view_ratio * toic_fused_w;
+  fused_toic_ += toic_evidence * in_view_ratio * toic_fused_w;
```

Comment:
error: no match for 'operator+=' (operand types are 'apollo::perception::fusion::Dst' and 'apollo::perception::fusion::Dst')

Reply:
My mistake, that's operator can not be overload in this way.